src/Plugin.NFC/Android/NFC.android.cs

```diff
@@ -94,21 +94,12 @@ public void StartListening()
 
 			var intent = new Intent(CurrentActivity, CurrentActivity.GetType()).AddFlags(ActivityFlags.SingleTop);
 
-			// We don't use MonoAndroid12.0 as targetframework for easier backward compatibility:
-			// MonoAndroid12.0 needs JDK 11.
-			PendingIntentFlags pendingIntentFlags = 0;
-			if ((int)Android.OS.Build.VERSION.SdkInt >= 31) //Android.OS.BuildVersionCodes.S
-				pendingIntentFlags = (PendingIntentFlags)33554432; //PendingIntentFlags.Mutable
```

Comment:
Are you sure this code can be removed? Could you please test to see if it working on Android 12 without these changes?

Reply:
I'm not sure. I don't have a device with android 12 for tests. I did the test with android 11 and it worked.